packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.base.tsx

```diff
@@ -838,7 +838,6 @@ export class ContextualMenuBase extends BaseComponent<IContextualMenuProps, ICon
 
     if (shouldHandleKey(ev)) {
       this._isFocusingPreviousElement = true;
-      ev.preventDefault();
```

Comment:
I don't believe this is the right fix, it should still prevent default to prevent the key press from propagating up and out. For instance a contextualmenu in a panel would dismiss the panel as well.